docs/ops/detection/RegionYolo_1.md

```diff
@@ -78,15 +78,18 @@
 
 **Inputs**:
 
-*   **1**: `data` - 4D input tensor with floating point elements and shape `[N, C, H, W]`. Required.
+*   **1**: `data` - 4D tensor of type `T` and shape `[N, C, H, W]`. **Required.**
 
 **Outputs**:
 
-*   **1**: output tensor of rank 4 or less that codes detected regions. Refer to the original YOLO paper to decode the output as boxes. `anchors` should be used to decode real box coordinates. If `do_softmax` is set to 0, then the output shape is `[N, (classes + coords + 1)*len(mask), H, W]`. If `do_softmax` is set to 1, then output shape is partially flattened and defined in the following way:
+*   **1**: tensor of type `T` and rank 4 or less that codes detected regions. Refer to the original YOLO paper to decode the output as boxes. `anchors` should be used to decode real box coordinates. If `do_softmax` is set to 0, then the output shape is `[N, (classes + coords + 1)*len(mask), H, W]`. If `do_softmax` is set to 1, then output shape is partially flattened and defined in the following way:
 
     flat_dim = data.shape[axis] * data.shape[axis+1] * ... * data.shape[end_axis]
     output.shape = [data.shape[0], ..., data.shape[axis-1], flat_dim, data.shape[end_axis + 1], ...]
```

Comment:
Can we wrap this text with some style modifier?

Reply:
Changed in this [commit](https://github.com/openvinotoolkit/openvino/pull/5926/commits/c0fd5d8d5baf158ea39d4f88df108dd61c530496).